openwisp-config/files/openwisp.agent

```diff
@@ -460,8 +463,8 @@ test_configuration() {
 }
 
 perform_default_test() {
-	# max 3 attempts to get checksum
-	for i in $(seq 1 3); do
+	# by default max 3 attempts to get checksum
+	for i in $(seq 1 $TEST_RETRIES); do
 		$($FETCH_COMMAND -i --connect-timeout 5 --max-time 5 $CHECKSUM_URL > $TEST_CHECKSUM)
 		local result=$?
 		if [ $result -gt 0 ]; then
```

Comment:
we should start logging failed attempts, it's useful to track these for debugging purposes:

Can you please add this before `sleep 5`?
```
logger "Configuration test attempt n. $i failed" \
       -t openwisp \
       -p daemon.warning
```

Reply:
Added.